src/run.ps1

```diff
@@ -51,16 +69,94 @@ function setPathForAzCliCert($config)
     }
 }
 
-function activate_venv()
-{
-    Write-Host "Activating python venv..."
-    .\.temp\.env\Scripts\activate.ps1
+function getLatestAzVersion() {
+    $gitUrl = "https://raw.githubusercontent.com/Azure/azure-cli/main/src/azure-cli/setup.py"
+    try {
+        $response = Invoke-WebRequest -Uri $gitUrl -TimeoutSec 30
+    }
+    catch {
+        logWarn "Failed to get the latest version from '$gitUrl': $($_.Exception.Message)"
+        return $null
+    }
+    if ($response.StatusCode -ne 200) {
+        logWarn "Failed to fetch the latest version from '$gitUrl' with status code '$($response.StatusCode)' and reason '$($response.StatusDescription)'"
+        return $null
+    }
+    $content = $response.Content
+    foreach ($line in $content -split "`n") {
+        if ($line.StartsWith('VERSION')) {
+            $match = [System.Text.RegularExpressions.Regex]::Match($line, 'VERSION = "(.*)"')
```

Comment:
Is this always expected to be present in the response content?
Is the connected vmware team also taking in this dependency?